cpp/src/hash/spark_murmurhash3_x86_32.cu

```diff
@@ -35,11 +36,11 @@ namespace {
 using spark_hash_value_type = int32_t;
 
 template <typename Key, CUDF_ENABLE_IF(not cudf::is_nested<Key>())>
-struct SparkMurmurHash3_32 {
+struct SparkMurmurHash3_x86_32 {
```

Comment:
Probably need an underscore after "Spark" to match the function name `spark_murmurhash3_x86_32`.
```suggestion
struct Spark_MurmurHash3_x86_32 {
```